nx.json

```diff
@@ -0,0 +1,33 @@
+{
+  "extends": "nx/presets/npm.json",
+  "pluginsConfig": {
+    "@nx/js": {
+      "analyzeSourceFiles": true
+    }
+  },
+  "targetDefaults": {
+    "lint": {
+      "inputs": [
+        "{workspaceRoot}/.eslintignore",
+        "{workspaceRoot}/.eslintrc.json",
+        "{workspaceRoot}/eslint.config.js",
```

Comment:
I can't find `eslint.config.js` file.

Reply:
You are correct! We can get rid of that line